packages/jasmine/jasmine_node_test.bzl

```diff
@@ -66,10 +67,9 @@ def jasmine_node_test(
         tags = [],
         config_file = None,
         use_direct_specs = None,
-        # Replaced by pkg_npm with jasmine = "//@bazel/jasmine",
-        jasmine = "//packages/jasmine",
-        # Replaced by pkg_npm with jasmine_entry_point = "//:node_modules/@bazel/jasmine/jasmine_runner.js",
-        jasmine_entry_point = Label("//packages/jasmine:jasmine_runner.js"),
+        # Kept for backward compatibility
```

Comment:
maybe a `TODO(6.0)` to remove it?

Reply:
Also, a new issue to remind us to remove it?

Reply:
not necessarily, we'll search for that string in the code when it's time